10.9-libcxx/stable/main/finkinfo/devel/meson-py.info

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8; tab-width: 4 -*-
+Info4: <<
+Package: meson-py%type_pkg[python]
+Version: 0.53.2
+Revision: 1
+Maintainer: Leigh Smith <leigh@leighsmith.com>
+Type: python (3.6 3.7 3.8)
+# setuptools needed because meson uses pkg_resources package.
+Depends: <<
+	python%type_pkg[python],
+	setuptools-tng-py%type_pkg[python]
+<<
+Source: https://files.pythonhosted.org/packages/49/d7/38a71de6adaa7a25a72587e69429ce78e137f4d545c7516341aa081f43e7/meson-%v.tar.gz
+Source-Checksum: SHA256(3e8f830f33184397c2eb0b651ec502adb63decb28978bdc84b3558d71284c21f)
+DocFiles: COPYING README.md
+CompileScript: <<
+ %p/bin/python%type_raw[python] setup.py build
+<<
+# Rename the installed files to match the python variant.
+InstallScript: <<
+ %p/bin/python%type_raw[python] setup.py install --root %d
+  mv %i/bin/meson %i/bin/meson-py%type_pkg[python]
+  mv %i/share/man/man1/meson.1 %i/share/man/man1/meson-py%type_pkg[python].1
+  mv %i/share/polkit-1/actions/com.mesonbuild.install.policy %i/share/polkit-1/actions/com.mesonbuild.install.policy-py%type_pkg[python]
+<<
+PostInstScript: <<
+  update-alternatives --install %p/bin/meson meson %p/bin/meson-py%type_pkg[python] %type_pkg[python] --slave %p/share/man/man1/meson.1 meson.1 %p/share/man/man1/meson-py%type_pkg[python].1 --slave %p/share/polkit-1/actions/com.mesonbuild.install.policy com.mesonbuild.install.policy %p/polkit-1/actions/com.mesonbuild.install.policy-py%type_pkg[python]
+<<
+PreRmScript: <<
+  if [ $1 != "upgrade" ]
+  then
+    update-alternatives --remove meson %p/bin/meson-py%type_pkg[python]
+  fi
+<<
+Description: Cross-platform build system
+DescDetail: <<
+Meson is an open source build system meant to be both extremely fast,
+and, even more importantly, as user friendly as possible. 
+
+The main design point of Meson is that every moment a developer
+spends writing or debugging build definitions is a second wasted.
+So is every second spent waiting for the build system to actually
+start compiling code. 
+<<
```

Comment:
DescPackaging: <<
```suggestion
<<
Ninja only needed for Ninja backend according to Homebrew recipe.
Skipping hotdoc build of markdown docs for now.
hdf5 tests are only run when installed, but need to be pointed to install loc;
for consistency made hdf5.100.v1.10 TestDepends.
Fixed some tests by setting python3 to the versioned executable, but some
failures remaining (e.g. tests trying to use Framework Python version etc.).
<<
```
If `meson` does not find a HDF5 installation, it will simply skip its `25 hdf` test, so an alternative would be to have the `PatchScript` search for either of the `hdf5.100.v1.10` or `hdf5.100.v1.8` include dirs and point to whatever it has found (or None), but that way tests would run differently depending on what packages are installed.